app/clients/sms/aws_pinpoint.py

```diff
@@ -32,6 +32,12 @@ def send_sms(self, to, content, reference, multi=True, sender=None, template_id=
         else:
             pool_id = self.current_app.config["AWS_PINPOINT_DEFAULT_POOL_ID"]
 
+        use_sc = str(template_id) in self.current_app.config["AWS_PINPOINT_SC_TEMPLATE_IDS"] or sending_vehicle == "short_code"
```

Comment:
I like that we keep the env var for now. Should we remove it long term? 🤔

Reply:
💯 yes - in this PR I'll bring in the feature flag and use that to do it the old way with `AWS_PINPOINT_SC_TEMPLATE_IDS` or the new way without it, basically.